.vib/moodle/cypress/cypress/integration/moodle_spec.js

```diff
@@ -0,0 +1,26 @@
+/// <reference types="cypress" />
+
+it('adds new course', () => {
+  cy.login();
+  cy.contains('a', 'Site administration').click();
+  cy.contains('a', 'Courses').click();
+  cy.contains('a', 'Add a new course').click();
+  cy.fixture('courses').then((courses) => {
+    cy.get('#id_fullname').type(courses.newCourse.fullName);
+    cy.get('#id_shortname').type(courses.newCourse.shortName);
+  });
+  cy.get('.fp-btn-add').click();
+  cy.contains('a', 'Upload a file').click();
+  cy.get('input[type="file"]').selectFile(
+    'cypress/fixtures/images/post_image.png',
```

Comment:
Could you please replace the used image with the one used for WordPress tests? That image will be the default one from now on.

Ref: [image](https://github.com/bitnami/charts/blob/master/.vib/wordpress/cypress/cypress/fixtures/images/test_image.jpeg)